Game/Scripts/Controller_FPSController.lua

```diff
@@ -3,6 +3,10 @@ local Controller_FPSController =
     currentWalkSpeed        = 1.0,
     currentRunSpeed         = 1.0,
     currentJumpStrength     = 1.0,
+    currentCameraFOV        = 55.0,
```

Comment:
Could we store the initial camera FOV on start? This way we would be able to still edit it from the inspector. Maybe the running fov should be a factor (1.2f?), so it works with any FOV set in the inspector.

We could lerp currentFov to targetFov, targetFov would be calculated each frame (initialFov if walking, and initialFov * scale if running), this way we only need to store these 2 variables:
- initialFov
- runFovFactor

Reply:
Yes let's rename that and change the runningFOV to a multiplier. We should also move the UpdateCameraFOV call in the `readyToGo` statement to avoid applying the FOV change when the player is inside the Cargo at the start of the game.